src/plonk/permutation/keygen.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) struct AssemblyHelper<C: CurveAffine> {
     deltaomega: Vec<Vec<C::Scalar>>,
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Since this is no longer used, we can remove it from this diff:
```suggestion
#[derive(Debug)]
```